EIC-EA/EIC-community-D8
# EICNET-904: Added storybook styling to group wiki pages.

Testing:

* Check the wiki section on the group edit page
* Create wiki (sub)pages

Problems:

* The levels of subpages is set to nine, so the add page below button is visible on the third level
* At the moment I only added the wiki page creator. There are no additional contributors.
* The hide sidebar button isn't working as on storybook, but i couldn't figure out why.
